lib/pipeline_controller.py

```diff
@@ -141,12 +151,16 @@ def process_broken_orbits(self, start_date, end_date):
         Re-queries individual sites with unhealthy data from the past 30 days (a limit
         set by the API) to see if any data has since been recovered
         """
+        found_sites_query = build_redshift_found_sites_query(
+            self.redshift_visits_table, start_date, end_date
+        )
         create_table_query = build_redshift_create_table_query(
             self.redshift_visits_table, start_date, end_date
         )
         self.redshift_client.connect()
+        found_site_dates = self.redshift_client.execute_query(found_sites_query)
         self.redshift_client.execute_transaction([(create_table_query, None)])
```

Comment:
Originally it was for executing a bunch of queries that depend on each other as part of the same transaction (e.g. don't delete rows in a table if the update to a different table fails). The naming is a bit of a relic though because now I mostly use it to do write queries (even outside of a transaction) whereas execute_query can only run read queries.